public/scripts/main.js

```diff
@@ -2,31 +2,95 @@ var __connected_ft = (function(){
 
 	'use strict';
 
-	var API_KEY = 'AAAARkLBNBk:APA91bFKrf-HAIZx7uVPP2fEOwLkoGi_EWpjuo9i-yMqg5Ub8Dg3p0MafUtloIvXDpZAJ48ZpftalWrPF7GgjebKYJf7zNiOmnu78CpTAXn2pdIEZ1oTd4JZnAibNarHOB_UNm1qOFGK';
+	var API_KEY = 'AAAARkLBNBk:APA91bG-5SRR12484VbE3rNlR1Xr2N0OunZNhq9YFpn38s_8mpXnqpCoTUc9MqF_qgrFitrc-pQox8pMb4C6RPprEp2KiAH7L3ET2los9em7n-6-hgAbjUOtpqudkrSvfzDFtw3tmJZh';
```

Comment:
Oh wait, I don't seem to use these at all... [I guess I'll remove them...](AAAARkLBNBk:APA91bGyHmXR04Qv3llISbLoQFTB6neGkN6i2QdELnG0ChSsbLcl20QZewSLQoGppQkvHLK07q4kxZxGHKieDisb6Vh5dsFqw1ZEp1SEVJSSnlBgFUsBwHF3bPJDzh_QSKEL0RVHERQR)